### ISSUE TITLE
Can't use 1.0.0 with my workflow

### ISSUE THREAD

**Issue by fooman** [magento/magento2-phpstorm-plugin#243]
**Is your feature request related to a problem? Please describe.**

Prior to version 1.0.0 I was able to use functionality of the plugin when developing extensions. The code is organised generally like this https://github.com/fooman/emailattachments-implementation-m2 with only the dependent Magento modules sitting under vendor (this is not a full Magento installation) after a composer install.

**Describe the solution you'd like** (*)

After upgrading to 1.0.0 I am required to supply a Magento path to be able to enable the plugin for a given project which I am unable to (the full install is provided via docker but is not necessarily required during dev, as for example unit tests can be run without it).

I understand that some functionality may require a full Magento instance. Maybe this can be optional so that 0.3.0 functionality like code navigation still works as that didn't require a path previously.

In summary could we make the validPath validation optional, and if it passes it unlocks the 1.0.0 features which depend on it. For now I will be sticking with 0.3.0.

**Comment by VitaliyBoyko** [comment]
@fooman 
Each module is dependent on the `magento/framework` right?

**Comment by fooman** [comment]
Thanks @VitaliyBoyko for looking into this. Yes I would consider `magento/framework` a requirement as otherwise it would not be a Magento specific extension but rather a library.

**Comment by VitaliyBoyko** [comment]
Cool, adjusting the root detection according to this. I will keep you posted.

**Comment by VitaliyBoyko** [comment]
Hi @fooman 
I've created an alpha version build with fix for your issue.
Could you please install it and check https://plugins.jetbrains.com/plugin/8024-magento-phpstorm/update/85848
Thank you!

**Comment by fooman** [comment]
@VitaliyBoyko that's great thank you. I can confirm that it installs and now let's me enable plugin for this project. Good stuff. Will keep this running and let you know if I come across anything odd.
